agent.go

```diff
@@ -543,9 +543,9 @@ func makeHTTPTransport(tlsConfig *tls.Config) *http.Transport {
 			tlsConn := tls.Client(tcpConn, tlsConfig)
 			return tlsConn, nil
 		},
-		// MaxIdleConns:        maxIdleConns,
-		// MaxIdleConnsPerHost: maxIdleConnsPerHost,
-		// IdleConnTimeout:     idleTimeout,
+		MaxIdleConns:        0,
```

Comment:
Real nitty nit: A comment here explaining what 0 means.

Reply:
MaxIdleConbns itself documents that.